pkg/config/protocol/instance.go

```diff
@@ -46,6 +46,12 @@ const (
 	Redis Instance = "Redis"
 	// MySQL declares that the port carries MySQL traffic.
 	MySQL Instance = "MySQL"
+	// DNS declares that the port carries DNS traffic.
+	DNS Instance = "DNS"
+	// Prometheus declares that the port carries Prometheus traffic.
+	Prometheus Instance = "Prometheus"
+	// Tiller declares that the port carries Tiller traffic.
+	Tiller Instance = "Tiller"
```

Comment:
I'm not 100% if it's ok to introduce these new protocols, however the reason for adding them is because galley forwards Synthetic Service entries to pilot which includes system generated service entries, without validation knowing about these protocols, pilot encounters a lot of `UnsupportedProtocol` warnings as part of protocol validation.

Reply:
I don't think we should add these, these are not protocols we support so `UnsupportedProtocol` is correct?

Reply:
This list seems arbitrary as well, is it just the port names that happen to be in your `kube-system`? On my system I have a different set

Reply:
@howardjohn that makes sense, I wasn't too sure if it's a right thing todo or not, I will update.